selfdrive/navd/map_renderer.h

```diff
@@ -38,13 +38,16 @@ class MapRenderer : public QObject {
   void publish(const double render_time, const bool loaded);
   void sendThumbnail(const uint64_t ts, const kj::Array<capnp::byte> &buf);
 
+  //QMapLibre::Style style;
   QMapLibre::Settings m_settings;
   QScopedPointer<QMapLibre::Map> m_map;
 
   void initLayers();
 
+  double start_render_t = 0, end_render_t = 0;
   uint32_t frame_id = 0;
   uint64_t last_llk_rendered = 0;
+  bool ok_render = true;
```

Comment:
can you give this a more descriptive name? had to read around to figure out what this was for